### PR TITLE
Base-types folder migration cleanup

### PR DESCRIPTION
**Pull request by tdykstra**
Moved changes from docs-internal; deleted earlier versions of base-types docs with truncated names, changed .NET Framework to .NET in newly migrated docs.  Another PR by @tompratt-AQ will do the redirects from the short names to the long names.

### CODE REVIEW

**Comment by tompratt-AQ** [comment]
[PR 1869](https://github.com/dotnet/docs/pull/1869) covers the redirects.

File: docs/standard/base-types/common-type-system.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ The common type system defines how types are declared, used, and managed in the
...
+ A structure is a value type that derives implicitly from <xref:System.ValueType?displayProperty=fullName>, which in turn is derived from <xref:System.Object?displayProperty=fullName>. A structure is very useful for representing values whose memory requirements are small, and for passing values as by-value parameters to methods that have strongly typed parameters. In .NET class library, all primitive data types (<xref:System.Boolean>, <xref:System.Byte>, <xref:System.Char>, <xref:System.DateTime>, <xref:System.Decimal>, <xref:System.Double>, <xref:System.Int16>, <xref:System.Int32>, <xref:System.Int64>, <xref:System.SByte>, <xref:System.Single>, <xref:System.UInt16>, <xref:System.UInt32>, and <xref:System.UInt64>) are defined as structures.
```
Comment (GuardRex) [inline docs/standard/base-types/common-type-system.md]: > In .NET class library, ...

In .NET class libraries, ...

or

In a .NET class library, ...

... and the comma splice in the previous sentence almost knocked me right out of my chair. :grinning:

File: docs/standard/base-types/trimming.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ author: "rpetrusha"
 ms.author: "ronpet"
 manager: "wpickett"
 ---
-# Trimming and Removing Characters from Strings  in the .NET Framework
+# Trimming and Removing Characters from Strings in the .NET Framework
```
Comment (GuardRex) [inline docs/standard/base-types/trimming.md]: Trimming and Removing Characters from Strings in .NET

**Review by guardrex — CHANGES_REQUESTED** [review]

**Comment by GuardRex** [comment]
@tdykstra I only see :eyes: two here. I wish I was that good. :+1:

File: docs/standard/base-types/trimming.md (hunk 2 above)
Comment (GuardRex) [inline docs/standard/base-types/trimming.md]: ... and the title as well. Did you leave this one alone on purpose?

**Comment by GuardRex** [comment]
@tdykstra Was the plan to catch-up on titles for TOC later? ... here's one, but there might be more lurking out there ...
```
### [.NET Framework Regular Expressions](regular-expressions.md)
```
Line 33 at https://github.com/dotnet/docs/edit/master/docs/standard/base-types/toc.md

**Comment by tdykstra** [comment]
@GuardRex No, I missed that.  I'll fix that and review toc.md,

File: docs/standard/base-types/trimming.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Trimming and Removing Characters from Strings  in the .NET Framework | Microsoft Docs"
+title: "Trimming and Removing Characters from Strings  in .NET | Microsoft Docs"
```
Comment (GuardRex) [inline docs/standard/base-types/trimming.md]: Nit: double-space before "in"

**Review by guardrex — APPROVED** [review]

File: docs/standard/base-types/trimming.md (hunk 3 above)
Comment (GuardRex) [inline docs/standard/base-types/trimming.md]: why do I do this ... genetic mutation? :alien: ... probably!

**Comment by GuardRex** [comment]
@tdykstra Thanks ... @wadepickett is going to either ❤️ me or 🔫 me for this, but I'm determined not to have @mairaw come back to a PR insanity! :grinning:

**Comment by tdykstra** [comment]
Thanks, @GuardRex, it'll be ❤️ not 🔫

Stale PRs are a breeding ground for merge conflicts.

**Comment by GuardRex** [comment]
> Stale PRs are a breeding ground for merge conflicts.

Tom, we're so on the same wavelength here ... @mairaw should come back to *FEWER* unmerged PR's than she left us with. For our kindred spirit, let's end the vacation on a high note 🎵 🎷.

**Pull request merged**